nstest.py

```diff
@@ -0,0 +1,18 @@
+"""Test that there is no core/arxiv/__init__.py
```

Comment:
I think it would be good (and not cheating) to exclude this file from coverage checks in `.coveragerc`.